tests/rptest/redpanda_cloud_tests/high_throughput_test.py

```diff
@@ -928,29 +934,116 @@ def topic_partitions_on_pod():
             f"Decommissioning node {pod_str}, partitions: {nt_partitions_before}"
         )
         decomm_time = time.monotonic()
-        admin = self.redpanda._admin
-        admin.decommission_broker(pod_id)
-        waiter = NodeDecommissionWaiter(self.redpanda,
-                                        pod_id,
-                                        self.logger,
-                                        progress_timeout=120)
-        waiter.wait_for_removal()
-        self.redpanda.stop_node(pod)
+
+        agent_services = [
+            'redpanda-agent.service', 'redpanda-agent-boot.service',
+            'redpanda-agent-init.service'
```

Comment:
note: `redpanda-agent-init.service` is only available in AWS, so disabling the non-existing service on GCP will fail